code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -255,60 +270,114 @@
 
 	if(isxenoqueen(X))
 		on_queen_death(X)
+	if(isxenoshrike(X))
```

Comment:
the hell is this shitcode doing here, should be done via overrides

Reply:
I know, but this is your code.